docs/content/en/docs/operator-manual/piped/installation.md

```diff
@@ -105,28 +105,24 @@ Most part is identical to the previous way, but some part is slightly different.
   apiVersion: pipecd.dev/v1beta1
   kind: Piped
   spec:
-    projectID: YOUR_PROJECT_ID
-    pipedID: YOUR_PIPED_ID
-    pipedKeyFile: PATH_TO_PIPED_KEY_FILE
+    projectID: {YOUR_PROJECT_ID}
```

Comment:
to keep all config file syntax consistent, I think all values should wrap in `{ }` or not at this time, so without `{ }` is better to me. We can make it by another PR which change all other configuration files 👀

Reply:
>I think all values should wrap in { } or not 

I feel exactly the same way. But I'm guessing the ones enclosed with `{ }` is better because there are a couple of fields to be all capitalized kind of like `KUBERNETES`, it can be confusing.

Reply:
Sure 👍 Then let's address it with another PR that refine all other configuration files 🙏

Reply:
Yep 👍